pkg/volume/plugins.go

```diff
@@ -319,6 +320,9 @@ type VolumeHost interface {
 	// Get mounter interface.
 	GetMounter(pluginName string) mount.Interface
 
+	// Get nsenter.
+	GetNsenter() *nsenter.Nsenter
```

Comment:
I would plumb this through some Mounter.ResolvePath call, instead of exposing nsenter

Reply:
Hmm yeah I’ll do that

Reply:
Yes, please add EvalHostSymlinks to mounter. I'd like to have all knowledge about containerized / non-containerized kubelet in mounter and nsenter, other components should have as little code branches for containerized case as possible.